SwiftAudio/Classes/AVPlayerWrapper/AVPlayerWrapper.swift

```diff
@@ -226,9 +228,8 @@ class AVPlayerWrapper {
         guard currentItem != nil else {
             throw APError.PlaybackError.noLoadedItem
         }
-        let millis = Int64(max(min(seconds, duration), 0) * 1000)
-        let time = CMTime(value: millis, timescale: 1000)
-        avPlayer.seek(to: time) { (finished) in
+        
+        avPlayer.seek(to: CMTimeMakeWithSeconds(seconds, 1)) { (finished) in
```

Comment:
I've changed this because there was a bug where if you seek immediately after playing, `duration` won't be set yet and therefore because of the `min()` makes seek always choose to seek to 0.

Later we can add a mechanism that holds the seek until `duration` is available but I think the "safety" we lose by removing this check isn't such a loss.